### ISSUE TITLE
Virtual Camera (streaming) instructions unclear

### ISSUE THREAD

**Issue by VACInc** [bambulab/BambuStudio#444]
In this link:
https://wiki.bambulab.com/en/software/bambu-studio/virtual-camera

It states:
```
If the ffmpeg tool is not installed, after confirming, start the download and installation. After the installation is complete, restart the streaming.

After the streaming starts (starting the ffmpeg streaming service for the first time), the sdp file location is prompted.
```
I am not able to tell if the ffmpeg is installed or where it is installed (the location) as it states here.

Also it states `In "Input", fill in: file:/ffmpeg.sdp (you can select the file without canceling the "local file" check box first, then copy the file path, add the "file:" prefix, and fill in here)` but I am unable to find the ffmpeg.sdp file.

**Comment by RobinWhitfield** [comment]
I'm suspect this is because the feature isn't arriving until the next release.

**Comment by bambu123** [comment]
That's right. This feature is not completed yet. The wiki page is public by accident.
